### PR TITLE
Adding the interface to the contracts

### PR DESCRIPTION
**Pull request by saurabh500**
Adding the interface IDbColumnSchemaGenerator to the contract for the DbDataReader schema

### CODE REVIEW

**Comment by saurabh500** [comment]
@kkurni

**Comment by kkurni** [comment]
LGTM

File: src/System.Data.Common/ref/System.Data.Common.Manual.cs [hunk 1]
```diff
@@ -79,4 +79,8 @@ public static bool CanGetColumnSchema(this DbDataReader reader)
         }
     }
 
+    public partial interface IDbColumnSchemaGenerator
```
Comment (weshaggard) [inline src/System.Data.Common/ref/System.Data.Common.Manual.cs]: It doesn't really matter but you can put this in the non-manual.cs file now. At this point none of the CS ref files are generated and thus they are all manual.

**Comment by saurabh500** [comment]
@dotnet-bot 
Test Innerloop OSX Release Build and Test please
Test Innerloop Ubuntu Debug Build and Test please 
Test Innerloop OSX Debug Build and Test please 
Test Innerloop CentOS7.1 Release Build and Test please
Test Innerloop CentOS7.1 Debug Build and Test please

**Pull request merged**